src/components/Organisms/Donate/Donate.style.js

```diff
@@ -48,7 +47,14 @@ const Wrapper = styled.div`
   align-items: center;
   display: block;
   width: 100%;
+  // TO-DO: this condition is WRONG and needs fixing (should be 'false'), but deploy this change will
+  // require a lot of content updates down the line in order to bring the pages back to the desired layout
```

Comment:
Gonna fix this in another more focussed PR so we're introducing too many moving parts at once, _just in case_